### PR TITLE
Update URL in Matreshka crates.

### PR DESCRIPTION
**Pull request by reznikmm**
The site https://forge.ada-ru.org isn't available any more, so replace source archives with GitHub release URLs.

### CODE REVIEW

File: index/ma/matreshka_amf/matreshka_amf-18.1.0.toml [hunk 1]
```diff
@@ -1,18 +0,0 @@
-description = "Implementation of OMG's Meta Object Facility (MOF)"
```
Comment (Fabien-Chouteau) [inline index/ma/matreshka_amf/matreshka_amf-18.1.0.toml:1]: Deletion of a crate release is not allowed. Why do you want to remove those releases?

**Review by Fabien-Chouteau — CHANGES_REQUESTED** [review]

File: index/ma/matreshka_amf/matreshka_amf-18.1.0.toml (hunk 1 above)
Comment (reznikmm) [reply index/ma/matreshka_amf/matreshka_amf-18.1.0.toml:1]: We lost `ada-ru.org` domain and 18.1 source archive.

Comment (reznikmm) [reply index/ma/matreshka_amf/matreshka_amf-18.1.0.toml:1]: There is an [archive in Fedora](https://src.fedoraproject.org/repo/extras/matreshka/) but sha512 is different. I don't understand why :(

Comment (Fabien-Chouteau) [reply index/ma/matreshka_amf/matreshka_amf-18.1.0.toml:1]: I will way on @mosteo to comment here, maybe the best solution is to mark the release a unavailable in all cases.

Comment (reznikmm) [reply index/ma/matreshka_amf/matreshka_amf-18.1.0.toml:1]: We have found a missing file (`matreshka-18.1.tar.gz`).

**Review by Fabien-Chouteau — APPROVED** [review]
Even if the builds are failing I am approving this since it's only changing URLs.

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
I guess this was submitted when we did fewer checks. Agreed with Fabien to go ahead since this preserves the current situation.

**Pull request merged**